src/components/SearchContainer/SearchContainer.css

```diff
@@ -0,0 +1,3 @@
+#search-container {
+  float: right;
```

Comment:
Let me know if you want me to take the time to implement a proper [`Flexbox` solution](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)